stubs/docker/docker/types/networks.pyi

```diff
@@ -3,29 +3,32 @@ from _typeshed import Incomplete
 class EndpointConfig(dict[str, Incomplete]):
     def __init__(
         self,
-        version,
-        aliases: Incomplete | None = None,
-        links: Incomplete | None = None,
-        ipv4_address: Incomplete | None = None,
-        ipv6_address: Incomplete | None = None,
-        link_local_ips: Incomplete | None = None,
+        version: str,
+        aliases: list[Incomplete] | None = None,
+        links: list[Incomplete] | None = None,
```

Comment:
`links` is passed to `normalize_links`, which can normalize a `dict` into a `list[str]`
```suggestion
        links: dict[str, str] | list[str] | None = None,
```